package.json

```diff
@@ -27,10 +27,10 @@
   "peerDependencies": {
     "bedrock": "2.0.0 - 4.x",
     "bedrock-jsonld-document-loader": "^1.0.1",
-    "bedrock-ledger-context": "^18.0.0"
+    "bedrock-ledger-context": "^20.0.0"
   },
   "engines": {
-    "node": ">=6.3.0"
+    "node": ">=14.0.0"
```

Comment:
Since this is a breaking change, I think we'll need a changelog that states that we're dropping support for node below v14 for this lib.

Reply:
+1 to what @JSAssassin is saying.  I will add however that we want to drop Node 12 support as a last resort.  Is it necessary to drop node 12 support?

Reply:
node 12 is not necessary was just trying to up the engine along with the upgrade. I believe `bedrock` is the only library this package depends on that has a node engine requirement. I set it to 12. https://github.com/digitalbazaar/bedrock-veres-one-context/pull/11/commits/f7c9b0cadee8d055a5c7aad5edad0458fb5b94ea

Reply:
@aljones15 TY for the clarification.  Please add a changelog entry for a breaking release.  It is breaking on account of this engine change as well as the change to the underlying context module.  Any major release context dependency changes in `bedrock-foo-context` modules like this one necessitate a major release in the bedrock module.